docs/Parameters.rst

```diff
@@ -240,14 +240,18 @@ Core Parameters
 
    -  refer to `Distributed Learning Guide <./Parallel-Learning-Guide.rst>`__ to get more details
 
--  ``num_threads`` :raw-html:`<a id="num_threads" title="Permalink to this parameter" href="#num_threads">&#x1F517;&#xFE0E;</a>`, default = ``0``, type = int, aliases: ``num_thread``, ``nthread``, ``nthreads``, ``n_jobs``
+-  ``num_threads`` :raw-html:`<a id="num_threads" title="Permalink to this parameter" href="#num_threads">&#x1F517;&#xFE0E;</a>`, default = ``None``, type = int, aliases: ``num_thread``, ``nthread``, ``nthreads``, ``n_jobs``
 
    -  used only in ``train``, ``prediction`` and ``refit`` tasks or in correspondent functions of language-specific packages
 
    -  number of threads for LightGBM
 
+   -  ``None`` means number of physical cores
+
```

Comment:
```suggestion
```

Let's remove this, in favor of a single new note for the `scikit-learn` interface.